### PR TITLE
Update GrayLoadingScreen

### PR DESCRIPTION
**Pull request by tyrosine1153**
### Description

[pledge 로딩 프로그레스 최신화](https://app.asana.com/0/958521740385861/1205688361705964/f)

- Edit main canvas initialize
  - move some first widgets to second widgets
  - move creating second widgets to before initializing second widgets from initializing agent
- Fix and Update GrayLoadingScreen Animation
  - delete screen animator
  - add loading slider handler animation 
- Update GrayLoadingScreen to call ShowProgress() in only Game,Start()

### CODE REVIEW

File: nekoyume/Assets/_Scripts/Game/Game.cs [hunk 1]
```diff
@@ -428,12 +429,13 @@ IEnumerator CoInitializeSecondWidget()
...
                 out var appProtocolVersion);
             Widget.Find<VersionSystem>().SetVersion(appProtocolVersion);
-            Widget.Find<GrayLoadingScreen>().ShowProgress(GameInitProgress.ProgressCompleted);
-            ShowNext(agentInitializeSucceed);
             Analyzer.Instance.Track("Unity/Intro/Start/ShowNext");
 
             StartCoroutine(CoUpdate());
             ReservePushNotifications();
+
+            yield return new WaitForSeconds(2f);
```
Comment (sonohoshi) [inline nekoyume/Assets/_Scripts/Game/Game.cs:437]: What is this waiting for?

File: nekoyume/Assets/_Scripts/UI/Widget/Screen/GrayLoadingScreen.cs [hunk 2]
```diff
@@ -79,7 +82,7 @@ public void ShowProgress(GameInitProgress progress)
             text.text = L10nManager.Localize($"UI_LOADING_GAME_START_{(int)progress}");
 
             loadingSlider.container.SetActive(true);
-            loadingSlider.slider.value = percent;
+            loadingSlider.slider.DOValue(percent, 2f);
```
Comment (sonohoshi) [inline nekoyume/Assets/_Scripts/UI/Widget/Screen/GrayLoadingScreen.cs:85]: https://github.com/planetarium/NineChronicles/pull/3213/files#r1363175601 same vibe?

**Review by sonohoshi — APPROVED** [review]

File: nekoyume/Assets/_Scripts/Game/Game.cs (hunk 1 above)
Comment (tyrosine1153) [reply nekoyume/Assets/_Scripts/Game/Game.cs:437]: it's waiting for last tweening in GrayLoadingScreen, when progress is 100

Comment (tyrosine1153) [reply nekoyume/Assets/_Scripts/Game/Game.cs:437]: I will add comment that description to code

File: nekoyume/Assets/_Scripts/UI/Widget/Screen/GrayLoadingScreen.cs (hunk 2 above)
Comment (tyrosine1153) [reply nekoyume/Assets/_Scripts/UI/Widget/Screen/GrayLoadingScreen.cs:85]: Ye

**Review by jonny-jeahyunchoi — APPROVED** [review]

**Pull request merged**